deploy/Mac_ARM-Delivery.yml

```diff
@@ -58,6 +58,15 @@ macOS_task:
     .ci/ci_controller.sh "$TARGET_ARCH" TEST
 
   package_script: |
+    otool -L run/john
+    if false; then
```

Comment:
A test package [1] was saved at: https://cirrus-ci.com/task/6572521295708160 (zip and 7z).

[1]  dynamic library saved the tarball (run folder)?